README.md

```diff
@@ -87,19 +90,9 @@ by a single command. Use `in-toto-record start ...` to create a
 preliminary link file that only records the *materials*, then run the
 commands of that step or edit files manually and finally use
 `in-toto-record stop ...` to record the *products* and generate the actual
-link metadata file.
-
-```shell
-usage: in-toto-record start --step-name <unique step name>
-                            (--key <signing key path> | --gpg [<gpg keyid>])
-                            [--gpg-home <gpg keyring path>] [-v]
-                            [--materials <material path> [<material path> ...]]
-
-usage: in-toto-record stop -step-name <unique step name>
-                           (--key <signing key path> | --gpg [<gpg keyid>])
-                           [--gpg-home <gpg keyring path>] [-v]
-                           [--products <product path> [<product path> ...]]
-```
+link metadata file. For a detailed list of all command line arguments and their usage,
+run `in-toto-record start --help` or `in-toto-record stop --help`. You can also
+find the [code documentation](https://github.com/in-toto/in-toto/blob/develop/in_toto/in_toto_record.py).
```

Comment:
This last sentence sounds a bit odd. Can you rephrase it to something like above or below.